sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_patch.py

```diff
@@ -0,0 +1,724 @@
+) -> Dict[str, Union[str, int]]:
+    return {
+        "id": response_headers["Schema-Id"],
+        "group_name": response_headers["Schema-Group-Name"],
+        "name": response_headers["Schema-Name"],
+        "version": int(response_headers["Schema-Version"]),
+    }
+
+
+def _get_format(content_type: str) -> SchemaFormat:
+    # pylint:disable=redefined-builtin
+    try:
+        format = content_type.split("serialization=")[1]
+        try:
+            format = SchemaFormat(format)
+        except ValueError:
+            format = SchemaFormat(format.capitalize())
+    except IndexError:
+        format = SchemaFormat.CUSTOM
+    return format
+
+
+def prepare_schema_properties_result(  # pylint:disable=unused-argument,redefined-builtin
+    format: str,
+    pipeline_response: "PipelineResponse",
+    deserialized: Union[Iterator[bytes], AsyncIterator[bytes]],
+    response_headers: Mapping[str, Union[str, int]],
+) -> Dict[str, Union[str, int]]:
+    properties_dict = _parse_schema_properties_dict(response_headers)
+    properties_dict["format"] = SchemaFormat(format)
+    pipeline_response.http_response.raise_for_status()
+    return properties_dict
+
+
+def prepare_schema_result(  # pylint:disable=unused-argument
+    pipeline_response: "PipelineResponse",
+    deserialized: Union[Iterator[bytes], AsyncIterator[bytes]],
+    response_headers: Mapping[str, Union[str, int]],
+) -> Tuple[Union["HttpResponse", "AsyncHttpResponse"], Dict[str, Union[int, str]]]:
+    properties_dict = _parse_schema_properties_dict(response_headers)
+    # TODO: content type is not being added to the response headers b/c of
+    properties_dict["format"] = _get_format(
+        cast(str, response_headers.get("Content-Type"))
+    )
+    pipeline_response.http_response.raise_for_status()
+    return pipeline_response.http_response, properties_dict
+
+
+###### Request Helper Functions ######
+
+def get_http_request_kwargs(kwargs):
+    http_request_keywords = ["params", "headers", "json", "data", "files"]
+    http_request_kwargs = {
+        key: kwargs.pop(key, None) for key in http_request_keywords if key in kwargs
+    }
+    return http_request_kwargs
+
+
+def get_content_type(format: str):  # pylint:disable=redefined-builtin
+    if format.lower() == SchemaFormat.CUSTOM.value.lower():
+        return "text/plain; charset=utf-8"
+    return f"application/json; serialization={format}"
+
+
+def get_case_insensitive_format(
+    format: Union[str, SchemaFormat]  # pylint:disable=redefined-builtin
+) -> str:
+    try:
+        format = cast(SchemaFormat, format)
+        format = format.value
+    except AttributeError:
+        pass
+    return format.capitalize()
+
+
+###### Wrapper Class ######
+
+class SchemaRegistryClient(object):
+    """
+    SchemaRegistryClient is a client for registering and retrieving schemas from the Azure Schema Registry service.
+
+    :param str fully_qualified_namespace: The Schema Registry service fully qualified host name.
```

Comment:
Is this different than the endpoint parameter? If not, I'm curious why we didnt use `@projectedName` in the typespec to generate this property name if that's the goal for Python.

Reply:
If this name is meant to be used by all generated sdks, then I would set the scope in `@projectedName` to `client`

Reply:
Same as the endpoint parameter. I just copied over what was in the swagger/other service typespecs, b/c it seems like using "endpoint" is the convention there. And I think we decided on fully_qualified_namespace here to follow SDK convention I think. I didn't know projectedName was a thing I could use here 🙂 will add it to the tsp!